nucypher/characters/lawful.py

```diff
@@ -1085,10 +1085,18 @@ def __init__(self,
 
     def __prune_arrangements(self) -> None:
         """Deletes all expired arrangements and kfrags in the datastore."""
-        now = datetime.fromtimestamp(self._arrangement_pruning_task.clock.seconds())
+        now = maya.MayaDT.from_datetime(datetime.fromtimestamp(self._arrangement_pruning_task.clock.seconds()))
         try:
-            result = self.datastore.del_expired_policy_arrangements(now=now)
-        except OperationalError:
+            with self.datastore.query_by(PolicyArrangement,
```

Comment:
I was thinking of making a `datastore.queries` module and moving a lot of these into that.

Since the datastore is accessed through a context manager and returns the `DatastoreRecord` objects, I wanted to have the module simply just be a bunch of `contextmanager` decorated functions that take a `datastore` as an argument. I did want to do this in a separate PR, though.